pkgs/tools/networking/ofono/default.nix

```diff
@@ -26,6 +26,8 @@ stdenv.mkDerivation rec {
     ./0001-Search-connectors-in-OFONO_PLUGIN_PATH.patch
   ];
 
+  postPatch = "substituteInPlace Makefile.am --replace 'dist_conf_DATA += plugins/phonesim.conf' ''";
```

Comment:
Does the following not work?

```nix
installFlags = [
  "SYSCONFDIR=${placeholder "out"}/etc"
];
```

Reply:
I'm afraid it might not work because ofono looks for `/nix/store/..../etc/plugins/phonesim.conf` by the package definition but I would like to use my own instead:
```
services.ofono.enable = true;
environment.etc."ofono/phonesim.conf".text = ''
  [phonesim]
  Driver=phonesim
  Address=127.0.0.1
  Port=1234
'';
```

Reply:
Yes, that is the point. Autotools will still use the `configureFlags` for hardcoding the path where it is looking but for the installation step, it will be overridden with the path from `installFlags`.

Reply:
@jtojnar It works. Thanks for giving the cleaner suggestion.

Reply:
pushed